experiments/views.py

```diff
@@ -125,3 +129,20 @@ def post(self, request, *args, **kwargs):
             log = ["form invalid"]
         return render(request, self.template_name, {'form': form,
                                                     'log': log})
+
+
+class DataView(View):
+    """
+    View to serve embeded metabase question
+    """
+
+    def get(self, request, *args, **kwargs):
+        payload = {
+            "resource": { "question": 1 },
+            "params": { "authorized_projects": None },
```

Comment:
minor PEP8 comments (I use auto-formatting a lot, so this will undergo unnecessary indentation changes from my side unless it's done by PEP8 from the very beginning):
- E201 - there should be no whitespace before/after "{"